linux_os/guide/system/software/integrity/crypto/configure_kerberos_crypto_policy/ansible/fedora.yml

```diff
@@ -0,0 +1,13 @@
+# platform = multi_platform_fedora
+# reboot = false
```

Comment:
Shouldn't this be rather `true`? If the configuration is already loaded linking the file won't reload it right?

Reply:
The service can be either restarted, or the system rebooted.
I have updated the remediation metadata in ansible fix, and added metadata to bash fix.